src/Cake.Squirrel/SquirrelAliases.cs

```diff
@@ -95,5 +95,74 @@ public static class SquirrelAliases
             var runner = new SquirrelRunner(context.FileSystem, context.Environment, context.ProcessRunner, context.Tools);
             runner.Run(nugetPackage, settings, new ProcessSettings { RedirectStandardOutput = redirectStandardOutput, Silent = silent });
         }
+
+        /// <summary>
+        /// Runs SyncReleases using the specified settings.
+        /// </summary>
+        /// <example>
+        /// <code>
+        /// #tool "Squirrel.Windows" 
+        /// #addin Cake.Squirrel
+        /// 
+        /// Task("SyncReleases")
+        ///  .Does(() => {
+        ///    var settings = new SyncReleasesSettings {
+        ///        ReleaseDirectory = "pathToDirectory"
+        ///        Url = new Uri("https://someurl.com");
+        ///        Token = "myToken"
+        ///    }; 
+        /// 
+        ///    SyncReleases(settings);
+        /// });
+        /// </code>
+        /// </example>
+        /// <param name="context">The context.</param>
+        /// <param name="settings">The settings.</param>
+        [CakeMethodAlias]
+        public static void SyncReleases(this ICakeContext context, SyncReleasesSettings settings)
```

Comment:
Cool! Now this makes me wonder if the methods called "Squirrel" should be refactored to "Releasify" to help reveal their intention when part of a build script.